src/components/dropdown_styles.css

```diff
@@ -0,0 +1,148 @@
+/* Dropdown Button */
+.dropbtn {
+display: fixed;
```

Comment:
we do not want this component to be fixed. This means that it will always stay in the position it is in even when we scroll. This is probably from copying from figma so be careful!